scripts/code_style.py

```diff
@@ -193,6 +204,10 @@ def main() -> int:
     parser.add_argument('-f', '--fix', action='store_true',
                         help=('modify source files to fix the code style '
                               '(default: print diff, do not modify files)'))
+    parser.add_argument('-g', '--git', action='store_true',
```

Comment:
Since this program requires git anyway (it requires `git ls-files`), why keep the non-git diff option?